Source/Fuse.Navigation/RoutePageProxy.uno

```diff
@@ -65,21 +65,23 @@ namespace Fuse.Navigation
 		}
 		List<Level> _levels = new List<Level>();
 		
+		/** `Init()` must be called to finish initialization. */
 		public RoutePageProxy( Visual source, ProgressUpdated progressUpdated )
 		{
 			_progressUpdated = progressUpdated;
-			Init(source);
+			_source = source;
 		}
 		
+		/** `Init()` must be called to finish initialization. */
 		public RoutePageProxy( Visual source, ActiveChanged activeChanged )
 		{
 			_activeChanged = activeChanged;
-			Init(source);
+			_source = source;
 		}
-		
-		void Init( Visual source )
+
+		/** Call after configuration is completed. This could start the actual registration and messaging */
+		public void Init()
```

Comment:
Doh. Then this API-change is no problem at all. In fact, it shouldn't even be mentioned in the changelog, as it isn't user-visible. Sorry for the noise ;)